docs/source/workflow.rst

```diff
@@ -0,0 +1,93 @@
+.. _ersemworkflow:
+
+##############
+ERSEM Workflow
+##############
+
+As a rule of thumb, ERSEM developments should be made in the `public ERSEM <https://github.com/pmlmodelling/ersem>`_
+with the exception of specific project developments.
+
+Issue tracking
+--------------
+
+All developments, either private or public, are required to create an 
+`issue/issues <https://github.com/pmlmodelling/ersem/issues>`_ describing the problem or
+development required.
+
+For larger project developments that require multiple issues, it is often preferable to create a
+`milestone <https://github.com/pmlmodelling/ersem/milestones>`_ which you can then assign multiple
+issues too.
+
+Private developments
+--------------------
+
+The only private developments would come from 
+specific projects - these require significant scientific changes to the model,
+and thus, are isolated from the public version of the code.
```

Comment:
I initially struggled to understand the sentence, maybe something like would be clearer
"Private developments are those generated within new and ongoing projects that requires significant scientific changes to the code that will be initially isolated from the public version of the code"

Reply:
Done